prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -107,6 +107,10 @@ class Hl7Serializer(val metadata: Metadata) {
             val parsedMessage = convertMessageToMap(nextMessage.toString(), schema)
             errors.addAll(parsedMessage.errors)
             warnings.addAll(parsedMessage.warnings)
+            // there is a chance that there's an empty row (for example, an empty line)
+            // that won't parse. so we should skip that because it's not valid HL7
+            if (parsedMessage.row.isEmpty())
+                return
             rowResults.add(parsedMessage)
```

Comment:
What it says. An empty line could cause problems with processing, so just ignore them.